<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidgetViewer</class>
 <widget class="QDockWidget" name="DockWidgetViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>524287</height>
   </size>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Viewer properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>209</y>
      <width>74</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle of view:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>182</width>
      <height>158</height>
     </rect>
    </property>
    <property name="title">
     <string>View geometry</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>128</y>
       <width>34</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>19</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Translation</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinViewRotX">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>38</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.099999999999994</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>127</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinViewRotY">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>63</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.099999999999994</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinViewRotZ">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>88</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.099999999999994</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinViewTransX">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>39</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinViewTransY">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>64</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinViewTransZ">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>88</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinViewScale">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>126</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBG">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>177</y>
      <width>49</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>172</y>
      <width>87</width>
      <height>32</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="text">
     <string>Brightness of background:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinAngleOfView">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>207</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>15</number>
    </property>
    <property name="maximum">
     <number>170</number>
    </property>
    <property name="value">
     <number>35</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinViewRotX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateRotX(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewRotY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateRotY(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewRotZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateRotZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewTransX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateTrans(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewTransY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateTrans(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewTransZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateTrans(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewScale</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateScale(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBG</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateBGcolor(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAngleOfView</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DockWidgetViewer</receiver>
   <slot>updateAngle(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateRotX(double)</slot>
  <slot>updateRotY(double)</slot>
  <slot>updateRotZ(double)</slot>
  <slot>updateTrans(double)</slot>
  <slot>updateScale(double)</slot>
  <slot>updateBGcolor(double)</slot>
  <slot>updateAngle(int)</slot>
 </slots>
</ui>
